wgsl/index.bs

```diff
@@ -952,8 +952,24 @@ array_decoration_list
 
 array_decoration
   : STRIDE PAREN_LEFT INT_LITERAL PAREN_RIGHT
+
+ident_type_decoration_list
+  : ATTR_LEFT (ident_type_decoration COMMA)* ident_type_decoration ATTR_RIGHT
+
+ident_type_decoration
+  : ACCESS PAREN_LEFT access_type PAREN_RIGHT
+
+access_type
+  : READ
+  | READ_WRITE
 </pre>
 
+The `access` decoration may only be applied to identifiers which are a
```

Comment:
I thought we had said these only apply to the store type for a storage buffer variable.

Reply:
So, should I say "[=host shareable=] in the `storage` [=storage class=]" or is just "in the `storage` [=storage class=]".

That's what I was trying to say with the host shareable, but realize that also includes uniforms which isn't right.

Reply:
I'd say

> The `access` decoration must only appear on a type used as the store type for a variable in the `storage` storage class.  The `access` decoration must not appear on a type of const declaration nor as the store type for variable with a storage class other than `storage`.

Then we can also eliminate the next two bullet points.

Somewhere the spec will define what a storage buffer variable is. But it doesn't exist yet. (!)

Maybe we should also so an example with a type alias as well.


```
type ROBuffer = [[access(read)]] Buffer;
var<storage> roBuf : ROBuffer; # Can read, cannot write
```

Reply:
Done.